examples/simple/src/posts/PostEdit.js

```diff
@@ -43,12 +45,25 @@ const EditActions = ({ basePath, data, hasShow }) => (
     </TopToolbar>
 );
 
+const SanitizedBox = ({ fullWidth, basePath, ...props }) => <Box {...props} />;
+
 const PostEdit = ({ permissions, ...props }) => (
     <Edit title={<PostTitle />} actions={<EditActions />} {...props}>
         <TabbedForm initialValues={{ average_note: 0 }} warnWhenUnsavedChanges>
             <FormTab label="post.form.summary">
-                <TextInput disabled source="id" />
-                <TextInput source="title" validate={required()} resettable />
+                <SanitizedBox
```

Comment:
why do you need that? It seems unrelated to the PR.

Reply:
It was a test to verify that we could now have anything inside a `FormTab` and the validation would still work. I can revert it if you want